src/fixed/method.rs

```diff
@@ -638,4 +749,50 @@ mod tests {
         );
         method.buffer.clear();
     }
+
+    #[test]
+    fn test_old_kar_order() {
+        let mut method = FixedMethod::default();
+        let mut config = get_fixed_method_defaults();
+        config.set_fixed_old_kar_order(true);
+
+        method.buffer = "".to_string();
+        method.process_key_value("ৈ", &config);
+        method.process_key_value("ক", &config);
+        assert_eq!(method.buffer, "কৈ".to_string());
+
+        method.buffer = "তে".to_string();
+        method.process_key_value("া", &config);
+        assert_eq!(method.buffer, "তো".to_string());
+
+        method.buffer = "সি".to_string();
+        method.process_key_value(&B_HASANTA.to_string(), &config);
+        method.process_key_value("ক", &config);
+        assert_eq!(method.buffer, "স্কি".to_string());
+
```

Comment:
Can you add this test case?:
```suggestion

       method.buffer = "".to_string();
       method.process_key_value("ি", &config);
       method.process_key_value("স", &config);
       method.process_key_value(&B_HASANTA.to_string(), &config);
       method.process_key_value("ট", &config);
       method.process_key_value("ম", &config);
       assert_eq!(method.buffer, "স্টিম".to_string());

```